politeiad/plugins/comments/comments.go

```diff
@@ -138,8 +157,16 @@ const (
 	// is found while comment plugin setting does not allow it.
 	ErrorCodeExtraDataNotAllowed = 12
 
+	// ErrorCodeEditsNotAllowed is returned when comment edits are not
+	// allowed.
+	ErrorCodeEditsNotAllowed = 13
+
+	// ErrorCodeExtraDataHintChangesNotAllowed is returned when an edited
+	// comment includes a different extra data hint than the existing one.
+	ErrorCodeExtraDataHintChangesNotAllowed = 14
```

Comment:
A generic "invalid edit" error, where the specific reason that it's invalid is included in the ErrorContext, might be more flexible here.  Up to you.

Reply:
yep, merged both error codes